lib/scope.js

```diff
@@ -2269,12 +2269,15 @@ module.exports = {
     let error_handlers = {};
     let was_found = false;
     for ( let error_key in possible_errors ) {
-      if ( possible_errors[ error_key ][0] ) {
+      if ( possible_errors[ error_key ][0]) {
+        let innerHTML = await possible_errors[ error_key][0].evaluate(node => node.innerHTML, possible_errors[error_key][0]);
+        if (innerHTML) {
         // If you're debugging and you close the window early, the process will still
         // take this long to exit after the tests finish. Not sure how to account for that.
         if ( session_vars.get_debug() ) { await scope.page.waitForTimeout( 60 * 1000 ); }
         was_found = true;
         error_handlers[ error_key ] = possible_errors[ error_key ][0];
+        }
```

Comment:
Code in new `if` needs indenting.

```suggestion
        if (innerHTML) {
          // If you're debugging and you close the window early, the process will still
          // take this long to exit after the tests finish. Not sure how to account for that.
          if ( session_vars.get_debug() ) { await scope.page.waitForTimeout( 60 * 1000 ); }
          was_found = true;
          error_handlers[ error_key ] = possible_errors[ error_key ][0];
        }
```